core/crypto/hasher.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_HASHER_HASHER_HPP_
+#define KAGOME_CORE_HASHER_HASHER_HPP_
+
+#include "common/blob.hpp"
+#include "common/buffer.hpp"
+
+namespace kagome::hash {
+  class Hasher {
+    using Hash128 = kagome::common::Hash128;
+    using Hash256 = kagome::common::Hash256;
+    using Buffer = kagome::common::Buffer;
```

Comment:
it is enough to write
```
using kagome::common::Buffer;
```

Reply:
core/crypto/hasher.hpp:18:9: error: using declaration in class refers into 'kagome::common::', which is not a class